src/Microsoft.HttpRepl.Tests/Microsoft.HttpRepl.Tests.csproj

```diff
@@ -6,11 +6,18 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Is this needed?

Reply:
Not required. Removed as part of this commit:
https://github.com/aspnet/HttpRepl/commit/e262cdca0ca93a629fd4dd674daaa799d3e199e5